src/main/dolphinManager.ts

```diff
@@ -0,0 +1,217 @@
+import { ChildProcessWithoutNullStreams, spawn } from "child_process";
+import { randomBytes } from "crypto";
+import { app } from "electron";
+import electronSettings from "electron-settings";
+import { EventEmitter } from "events";
+import * as fs from "fs-extra";
+import { find, remove } from "lodash";
+import path from "path";
+
+import { DolphinType } from "../renderer/lib/directories";
```

Comment:
It's important that we don't mix up things that are imported in `main` and things that are imported in `renderer`. If it's something that's used by both, we should move it to `common`.